addons/l10n_latam_invoice_document/wizards/account_move_reversal.py

```diff
@@ -22,6 +21,8 @@ def default_get(self, fields):
             move_ids_use_document = move_ids.filtered(lambda move: move.l10n_latam_use_documents)
             if move_ids_use_document:
                 raise UserError(_('You can only reverse documents with legal invoicing documents from Latin America one at a time.\nProblematic documents: %s') % ", ".join(move_ids_use_document.mapped('name')))
+        else:
+            res['l10n_latam_use_documents'] = move_ids.journal_id.l10n_latam_use_documents
```

Comment:
any([m.journal_id.l10n_latam_use_documents for m in move_ids]) ?

Reply:
```python3
if len(move_ids) > 1:
    ...
else:  # len(move_ids) <= 1
    ...

Reply:
I don't get it?

Reply:
The size of move_ids is maximum 1 because of the `if ... else ...` block. So doing a `any()` is useless.
And if the size is 0, then it will be `False` by default, same result as `any([])`

Note: wrapping the generator in a list (`any([generator])`) is bad practice because you force python to first compute the value of `m.journal_id.l10n_latam_use_documents` for every move in `move_ids` instead of stopping at the first one. Doing `any(generator)` instead is better.

Reply:
indeed, I didn't see the` if... else...` statement.

Note on the note: that remark would stand if the list comprehension was returning a generator, but it's not.

Reply:
Meh
```python3
Python 3.6.4 (default, Nov  5 2018, 12:04:37) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.9.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: l = env['account.move'].search([])                                      

In [2]: print(r for r in l)                                                     
<generator object <genexpr> at 0x7fca48044f10>
```